src/sentry/audit_log/manager.py

```diff
@@ -82,21 +80,20 @@ class AuditLogEventManager:
     def __init__(self) -> None:
         self._event_registry = {}
         self._event_id_lookup = {}
+        self._api_name_lookup = {}
 
     def add(self, audit_log_event: AuditLogEvent):
         if (
             audit_log_event.name in self._event_registry
             or audit_log_event.event_id in self._event_id_lookup
         ):
-            capture_exception(
-                DuplicateAuditLogEvent(
-                    f"Duplicate audit log: {audit_log_event.name} with ID {audit_log_event.event_id}"
-                )
+            raise DuplicateAuditLogEvent(
```

Comment:
@mbauer404 You'll want to raise a duplicate error if `self._api_name_lookup[audit_log_event.api_name]` already exists here.